src/components/Modal.js

```diff
@@ -0,0 +1,94 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+import { connectTheme } from '../utils';
+import { Header } from '.';
+
+class Modal extends PureComponent {
+  static get propTypes() {
+    return {
+      sytle: PropTypes.object,
```

Comment:
typo

Reply:
thanks. also added styles for each part of the component